quartz/components/ExplorerNode.tsx

```diff
@@ -124,9 +125,10 @@ export class FileNode {
    * Get folder representation with state of tree.
    * Intended to only be called on root node before changes to the tree are made
    * @param collapsed default state of folders (collapsed by default or not)
+   * @param currentFile current file
    * @returns array containing folder state for tree
    */
-  getFolderPaths(collapsed: boolean): FolderState[] {
+  getFolderPaths(collapsed: boolean, currentFile: string): FolderState[] {
```

Comment:
doesn't seem like we are using this anywhere in the function?

Reply:
Fixed.